docs/pages/Cardinal/CoreConcepts/Systems.mdx

```diff
@@ -0,0 +1,88 @@
+# Systems
+
+Systems are the functions that are run every tick. Systems must adhere to the following function signature:
+
+```go
+import "github.com/argus-labs/world-engine/cardinal/ecs"
+
+func SomeSystem(*ecs.World, *ecs.TransactionQueue) error
+```
+
+Systems give you a reference to the world, as well as the transaction queue. This allows you to access any transaction currently queued for this tick, as well as update any component state.
+
+## Getting Transactions
+
+In order to get specific transactions from the queue, utilize the transaction's `In` method. This transaction method takes the queue as an argument, and returns a slice of `TxData`, which contains data related to that transaction.
+
+```go
+import "github.com/argus-labs/world-engine/cardinal/ecs"
+
+func SomeSystem(world *ecs.World, queue *ecs.TransactionQueue) error {
```

Comment:
I prefer examples that have more concrete naming so that it provides better illustration to the reader. Instead of `SomeSystem` it should be `AttackSystem` and instead of `SomeTx` it should be `AttackTx`